avocado/core/loader.py

```diff
@@ -613,7 +613,7 @@ def _find_avocado_tests(self, path):
         # The name used, in case of 'import avocado as avocadolib'
         mod_import_name = None
         # The resulting test classes
-        result = {}
+        result = collections.OrderedDict()
```

Comment:
First reason is people should not depend on the order of tests (they sometimes do and that is really dangerous). Second reason is it's not compatible with unittest, which can be confusing for users.